github/authorizations.go

```diff
@@ -274,10 +274,14 @@ func (s *AuthorizationsService) Delete(ctx context.Context, id int64) (*Response
 // The returned Authorization.User field will be populated.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#check-an-authorization
-func (s *AuthorizationsService) Check(ctx context.Context, clientID string, token string) (*Authorization, *Response, error) {
-	u := fmt.Sprintf("applications/%v/tokens/%v", clientID, token)
+func (s *AuthorizationsService) Check(ctx context.Context, clientID string, clientToken string, accessToken string) (*Authorization, *Response, error) {
```

Comment:
s/clientID string, clientToken string, accessToken string/clientID, clientToken, accessToken string/

Please also add a comment about the difference between `clientToken` and `accessToken` and how to know which is for what purpose.